### DISCUSSION TITLE
Carla with Autoware.auto

### DISCUSSION THREAD

**Discussion by tecena** [carla-simulator/carla#5486] [Q&A]
Hi,
I found that carla runs fine with various versions of Autoware.ai versions,
but keen to know if Carla runs with Autoware.auto.
Does Carla supports Autoware.auto
Thank you

**Comment by ralwing** [comment]
See here:
https://github.com/orgs/autowarefoundation/discussions/420
And here:
https://github.com/carla-simulator/ros-bridge
